app/views/posts/index.html.erb

```diff
@@ -2,30 +2,7 @@
 
 <h1>Posts</h1>
 
-<table>
-  <thead>
-    <tr>
-      <th>Content</th>
-      <th>Grass tags</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-
-<!-- update to hide posts that are 1) only_followers && 2) the current_user is not following.  Also update so edit and destroy are only availalbe to the user whose post it is -->
-
-    <% @posts.reverse.each do |post| %>
-      <tr>
-        <td><%= post.content %></td>
-        <td><%= post.grass_tags %></td>
-        <td><%= link_to 'Show', post %></td>
-        <td><%= link_to 'Edit', edit_post_path(post) %></td>
-        <td><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+  <%= render "shared/post_rendering" %>
```

Comment:
Rendering the Posts in this way turned out to be identical to the way they're being presented on the `user/dashboard`, so I made it into a shared file